mlir/include/mlir/Dialect/DLTI/DLTI.h

```diff
@@ -55,6 +55,10 @@ class DataLayoutEntryAttr
 
   /// Prints this attribute.
   void print(AsmPrinter &os) const;
+
+  static constexpr StringRef getAttrName() {
```

Comment:
Why not just `getName()` so this is the same API on Types and Attribute (which is always more friendly to generic code).

Reply:
Reasonnable I guess